Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -214,14 +214,18 @@ public enum GeneralTraitsOptions {
 May produce spurious warnings.") {
     IsHidden = true
   };
-  public static readonly Option<string> VerificationCoverageReport = new("--coverage-report",
-    "Emit verification coverage report  to a given directory, in the same format as a test coverage report.") {
+  public static readonly Option<string> VerificationCoverageReport = new("--verification-coverage-report",
+    "Emit verification coverage report to a given directory, in the same format as a test coverage report.") {
     ArgumentHelpName = "directory"
   };
   public static readonly Option<bool> NoTimeStampForCoverageReport = new("--no-timestamp-for-coverage-report",
     "Write coverage report directly to the specified folder instead of creating a timestamped subdirectory.") {
     IsHidden = true
   };
+  public static readonly Option<string> ExecutionCoverageReport = new("--coverage-report",
```

Comment:
You could call this `--execution-coverage-report`, since that term shows up in the variable name and the description text. :)

Reply:
I did consider that, but then it would feel weird to not further qualify `dafny generate-tests --coverage-report` as well, perhaps as `--expected-execution-coverage-report`. Given the `generate-tests` one is already released I'm more inclined to leave both with the simpler name.